GitUI/CommandsDialogs/BrowseDialog/FormUpdates.cs

```diff
@@ -88,23 +98,44 @@ private void SearchForUpdates()
 
         }
 
-        void CheckForNewerVersion(string releases)
+        void CheckForNewerVersion(GitHubReleaseInfo release)
         {
-            var versions = ReleaseVersion.Parse(releases);
-            var updates = ReleaseVersion.GetNewerVersions(CurrentVersion, AppSettings.CheckForReleaseCandidates, versions);
```

Comment:
http://www.smarterasp.net/hosting_plans


On Sun, Jul 17, 2016 at 3:19 PM Jay Asbury <vbjay.net@gmail.com> wrote:

> We should have a webservice that returns the feed url.  This way the
> serving is dynamic and the host can be changed.  Gitextensions.org is
> available
>
> On Sun, Jul 17, 2016 at 2:49 PM Janusz Białobrzewski <
> notifications@github.com> wrote:
>
>> In GitUI/CommandsDialogs/BrowseDialog/FormUpdates.cs
>> <https://github.com/gitextensions/gitextensions/pull/3232#discussion_r71087224>
>> :
>>
>> >          {
>> > -            var versions = ReleaseVersion.Parse(releases);
>> > -            var updates = ReleaseVersion.GetNewerVersions(CurrentVersion, AppSettings.CheckForReleaseCandidates, versions);
>>
>> What about CheckForReleaseCandidates?
>>
>> —
>> You are receiving this because you are subscribed to this thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/gitextensions/gitextensions/pull/3232/files/9ed3aa97e7dfe892553c6cfbfc69fac088ddd9b4#r71087224>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/ADdhsQ5fs_R2dw__RDwvIvFrlkiIdaIOks5qWnkjgaJpZM4JD1n_>
>> .
>>
>